R/opts.R

```diff
@@ -201,11 +130,20 @@ delay_opts <- function(..., fixed = FALSE) {
 #' trunc_opts()
 #'
 #' # truncation dist
-#' trunc_opts(dist = list(mean = 3, sd = 2))
-trunc_opts <- function(dist = list()) {
-  data <- do.call(dist_spec, dist)
+#' trunc_opts(dist = dist_spec(mean = 3, sd = 2, max = 10))
+trunc_opts <- function(dist = dist_spec()) {
+  if (!"dist_spec" %in% class(dist)) {
```

Comment:
shared code with delay_opts and generation_time_opts - could be a helper function

Reply:
Done in https://github.com/epiforecasts/EpiNow2/pull/363/commits/bf455251037ccea9f31ec2196fc9304a3e218d89